tests/waterservices_test.py

```diff
@@ -82,7 +82,7 @@ def test_get_iv(requests_mock):
     site = '01491000%2C01645000'
     request_url = 'https://waterservices.usgs.gov/nwis/iv?format={}' \
                   '&startDT=2019-02-14&endDT=2020-02-15&sites={}'.format(format, site)
-    response_file_path = 'data/waterservices_iv.txt'
+    response_file_path = 'tests/data/waterservices_iv.txt'
```

Comment:
Why is `tests/` put before each response_file_path?  That path doesn't exist relative to this file and all the tests will fail.

Reply:
That's for the github action, which runs pytest from the dataretrieval directory by default. Not sure how to change that behavior.

Reply:
I suppose one would cd to 'tests' during the action. Is that your preference?

Reply:
I would prefer the change directory.  That'll allow us to run tests individually from an IDE.

Reply:
Actually, I can change the directory it runs from in my IDE.  I'm fine either way, whichever you think is best.